schema/plugin.json

```diff
@@ -20,7 +20,12 @@
       "title": "Hide Cluster Manager",
       "description": "Some deployments don't want to or are unable to use the cluster manager feature. Toggle to hide it from the user interface (note: this does not disable the underlying functionality).",
       "default": false
-    }
+    },
+    "enableTestDashboardFetch": {
+      "type": "boolean",
+      "title": "Enable Test Dashboard with Fetch",
+      "description": "If set to true, the test dashboard will be validated with the a simple fetch request. This is useful for testing the dashboard when behind authentication.",
```

Comment:
As above, the important thing is that it's a browser-side request, rather than `fetch`. I'm not 100% familiar with the QHub setup, but would it be correct to say that it's useful when a browser cookie is needed for authentication? (rather than just any authentication, which might also happen on the server side)

Reply:
I updated the descriptions, that's definitely the case. Thanks for pointing that out.